pkg/storaged/vdos-panel.jsx

```diff
@@ -141,7 +141,8 @@ export class VDOsPanel extends React.Component {
                                       name={vdo.name}
                                       detail={fmt_size(vdo.logical_size)}
                                       go={() => cockpit.location.go([ "vdo", vdo.name ])}
-                                      job_path={block && block.path} />
+                                      job_path={block && block.path}
+                                      key={vdo} />
```

Comment:
`vdo` is not a string, is that a valid value for a `key` property?

Reply:
AFAIK, yes - any object should be ok. Reference to the object (not deep content) is compared for equality.

Reply:
Ok, thanks.

Reply:
Unfortunately, it's not okay: https://stackoverflow.com/questions/28982336/reactjs-key-attribute-type

`vdo.dev` should work.

Reply:
Fixed in "storage: More React keys"